corehq/toggles.py

```diff
@@ -87,7 +88,7 @@ def index(self):
 TAG_SOLUTIONS_LIMITED = Tag(
     name='Solutions - Limited Use',
     css_class='info',
-    description=mark_safe(
+    description=mark_safe(  # nosec: no user input
```

Comment:
is there a reason this can't be switched to `format_html`?

Reply:
It could be, but it didn't seem like there was a real benefit to doing so. We're not building up a string here, as it's already all together. If we decided we wanted to translate this string, I'd suggest extracting out the URL to a separate parameter, and at that point, `format_html` would make more sense.